tests/e2e/e2e_test.go

```diff
@@ -73,6 +73,35 @@ func pingSvc(url string) {
 	}
 }
 
+func waitForCmdOut(cmd string, expOut string) bool {
+	// This function runs the command until it \
```

Comment:
The comment should be before the function.
Check this out https://blog.golang.org/godoc-documenting-go-code